layer/compile_time_layer.cc

```diff
@@ -35,10 +36,23 @@ constexpr char kLayerDescription[] =
     "Stadia Pipeline Compile Time Measuring Layer";
 constexpr char kLogFilenameEnvVar[] = "VK_COMPILE_TIME_LOG";
 
-class CompileTimeLayerData : public LayerData {
+class CompileTimeEvent : public Event {
+ public:
+  CompileTimeEvent(const char* name, std::vector<int64_t>& hash_values,
+                   int64_t duration)
```

Comment:
Shouldn't we have a separate attribute for duration? E.g., `DurationNsAttr`. This way it's obvious what the unit is.